source/sass/patterns/molecules/figure.scss

```diff
@@ -0,0 +1,19 @@
+@import "../../mixins/spacing";
+@import "../../mixins/typography";
+
+.figure {
+  @include block-spacing($end: $baselinegrid-space-medium);
+  @include figure-caption-typography();
+}
+
+.figure__image {
+  @include block-spacing($end: $baselinegrid-space-extra_small);
+  @include margin(12px, inline);
+  border: 1px solid $color-text-dividers;
+}
+
+.figure__heading {
+  @include figure-caption-typography();
+  @include nospace();
+  font-weight: bold;
```

Comment:
Shouldn't be needed, headings are always semi-bold (and we haven't been including a bold font).